packages/ui-components/src/components/description-list/description-list.config.ts

```diff
@@ -0,0 +1,7 @@
+import { ComputePositionConfig, offset } from '@floating-ui/dom';
+import { ETooltipPosition } from '../../types';
+
+export const TEXT_TOOLTIP_CONFIG: Partial<ComputePositionConfig> = {
+	placement: ETooltipPosition.Bottom,
+	middleware: [offset(4)]
```

Comment:
Do you think we should have this 4 in a config to be more understanding what kind of offset we are applying here? Top/bottom?